setup.py

```diff
@@ -24,11 +28,7 @@ def initialize_options(self):
 
     def run(self):
         if self.build:
-            if self.build.startswith('+'):
-                prefix = ''
-            else:
-                prefix = '.'
-            self.distribution.metadata.version += f'{prefix}{self.build}'
+            self.distribution.metadata.version += f'{self.build}'
```

Comment:
I am not sure why there is no longer a need to handle a prefix to the build version, can you elaborate?

Reply:
I have changed this back